### PR TITLE
google analytics

### PR DESCRIPTION
**Pull request by saphal-upreti**
setup google analytics for nepalingo website

### CODE REVIEW

File: nepalingo-web/package-lock.json
Comment (christikaes) [inline nepalingo-web/package-lock.json:1]: Looks like you're on an older version of PNPM - can you pls update to the latest?

File: nepalingo-web/index.html
Comment (christikaes) [inline nepalingo-web/index.html:17]: This will work but I think it'll be easier for us if we use a react library for this: https://medium.com/@dilhancodes/how-to-install-google-analytics-in-react-project-in-2024-205759218057
https://www.npmjs.com/package/react-ga4

This will provide some useful utility functions so that we can start adding tracking events.

**Review by christikaes — CHANGES_REQUESTED** [review]
Let's switch to using a library package

File: nepalingo-web/package-lock.json
Comment (saphal-upreti) [reply nepalingo-web/package-lock.json:1]: ohh okay

File: nepalingo-web/src/App.tsx
Comment (christikaes) [reply nepalingo-web/src/App.tsx:14]: can be done later

**Review by christikaes — COMMENTED** [review]
Looks like you still have a lint issue that will block merging - once that's done I'll approve!

**Comment by binamkayastha** [comment]
Some of these lint issues is not Saphals fault. They were caused by another PR that was merged to main. I updated main and merged the main branch into this one and now the lints pass (I ran `git merge main` after updating my main branch)

**Pull request merged**
